tests/wpt/include-layout-2020.ini

```diff
@@ -13,6 +13,8 @@ skip: true
       skip: false
     [box-display]
       skip: false
+    [generated-content]
+      skip: false
     [zindex]
       skip: false
     [zorder]
```

Comment:
Let’s also enable `web-platform-tests/css/css-content` here, even if we don’t yet implement anything from https://drafts.csswg.org/css-content/. It’s possible that some of those tests cover some of the functionality from CSS2.